wlutil/launch.py

```diff
@@ -75,16 +113,16 @@ def getQemuCmd(config, nodisk=False):
     return " ".join(cmd) + " " + config.get('qemu-args', '')
 
 
-def launchWorkload(baseConfig, jobs=None, spike=False, interactive=True):
+def launchWorkload(baseConfig, jobs=None, spike=False, silent=False, captureOutput=True):
     """Launches the specified workload in functional simulation.
 
     cfgName: unique name of the workload in the cfgs
     cfgs: initialized configuration (contains all possible workloads)
     jobs: List of job names to launch. If jobs is None we use the parent of
     all the jobs (i.e.  the top-level workload in the config).
     spike: Use spike instead of the default qemu as the functional simulator
-    interactive: If true, the output from the simulator will be displayed to
-        stdout. If false, only the uartlog will be written (it is written live and
+    silent: If false, the output from the simulator will be displayed to
+        stdout. If true, only the uartlog will be written (it is written live and
         unbuffered so users can still 'tail' the output if they'd like).
```

Comment:
add captureOutput description to comment

Reply:
Done!